### ISSUE TITLE
Uncaught TypeError: Cannot read property 'absolutePath' of undefined

### ISSUE THREAD

**Issue by Paul404** [mulesoft/api-workbench#87]
[Enter steps to reproduce below:]

1. part of my raml:  
    response:  
       200:  
           displayName: test  
           description: blah blah blah  
2. Error occurred when clicked "body" in Details tab.

**Atom Version**: 1.3.2
**System**: Unknown Windows Version
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.11

### Stack Trace

Uncaught TypeError: Cannot read property 'absolutePath' of undefined

```
At C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:3

TypeError: Cannot read property 'absolutePath' of undefined
    at t.typeId (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:3:6296)
    at t.allProperties (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:10:27117)
    at t.allProperties (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:3:5539)
    at e.process (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:20:16826)
    at e.process (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:20:16307)
    at t.children (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:3:28606)
    at t.attrs (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:3:30500)
    at t.attr (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:3:30139)
    at C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:7:18248
    at Array.forEach (native)
    at e.createStubNode (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:7:18169)
    at e.insert (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:31:23399)
    at C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:31:22500
    at C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:27:12162
    at Array.forEach (native)
    at HTMLButtonElement._onClickListeners.length._onAltClickListeners.length.e.onclick (C:\Users\jingjong.chen\.atom\packages\api-workbench\main.js:27:12135)
```

### Commands

```
  4x -5:04.0 core:backspace (atom-text-editor.editor.is-focused)
     -5:02.6.0 core:paste (atom-text-editor.editor.is-focused)
     -4:58 snippets:previous-tab-stop (atom-text-editor.editor.is-focused)
     -4:58 editor:outdent-selected-rows (atom-text-editor.editor.is-focused)
     -4:57.4.0 snippets:previous-tab-stop (atom-text-editor.editor.is-focused)
     -4:57.4.0 editor:outdent-selected-rows (atom-text-editor.editor.is-focused)
     -4:56.9.0 snippets:previous-tab-stop (atom-text-editor.editor.is-focused)
     -4:56.9.0 editor:outdent-selected-rows (atom-text-editor.editor.is-focused)
     -4:55.3.0 snippets:previous-tab-stop (atom-text-editor.editor.is-focused)
     -4:55.3.0 editor:outdent-selected-rows (atom-text-editor.editor.is-focused)
     -4:41.8.0 core:save (atom-text-editor.editor.is-focused)
     -4:07.9.0 editor:newline (atom-text-editor.editor.is-focused)
  3x -4:04.0 core:backspace (atom-text-editor.editor.is-focused)
     -3:54 editor:newline (atom-text-editor.editor.is-focused)
     -1:51.7.0 api-workbench:Create new Response (atom-text-editor.editor.is-focused)
  3x -1:21 core:backspace (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {},
  "api-workbench": {}
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.11
linter, v1.11.3

# Dev
No dev packages
```

**Comment by ddenisenko** [comment]
Reproducible. Thank you @Paul404 for reporting it.

Steps: 
1) RAML:

```RAML
#%RAML 1.0
title: test

/helloWorld:
  get:
    responses:
      200:
        displayName: test
        description: blah blah

```

2) Put cursor at `200:`, open Palette, click `body` entry in the Palette.

**Comment by ddenisenko** [comment]
Setting to High as it looks like the issue happens more often than just with `body`.

**Comment by ddenisenko** [comment]
Fix should be available in 0.8.12.
